bridges/FacebookBridge.php

```diff
@@ -659,16 +659,10 @@ private function collectUserData(){
 							$date = 0;
 						}
 
-						// Build title from username and content
-						$title = $author;
-
-						if(strlen($title) > 24)
-							$title = substr($title, 0, strpos(wordwrap($title, 24), "\n")) . '...';
-
-						$title = $title . ' | ' . strip_tags($content);
-
+						// Build title from content
+						$title = strip_tags($post->find('.userContent')[0]->innertext);
 						if(strlen($title) > 64)
-							$title = substr($title, 0, strpos(wordwrap($title, 64), "\n")) . '...';
+							$title = substr($title, 0, 64) . '...';
```

Comment:
```suggestion
							$title = substr($title, 0, strpos(wordwrap($title, 64), "\n")) . '...';
```

That way we don't split titles mid-word. This code will find the longest sentence with maximum 64 characters.